connector/connect/server/src/main/scala/org/apache/spark/sql/connect/execution/ExecuteGrpcResponseSender.scala

```diff
@@ -34,7 +35,7 @@ import org.apache.spark.sql.connect.service.ExecuteHolder
  * @param grpcObserver
  *   the GRPC request StreamObserver
  */
-private[connect] class ExecuteGrpcResponseSender[T <: MessageLite](
+private[connect] class ExecuteGrpcResponseSender[T <: Message](
```

Comment:
What's the reason for this particular change?

Reply:
`ProtoUtils.abbreviate` wants a `Message`